testsuite/features/core/srv_create_activationkey.feature

```diff
@@ -119,3 +119,15 @@ Feature: Be able to create and manipulate activation keys
     And I select "Push via SSH" from "contact-method"
     And I click on "Create Activation Key"
     Then I should see a "Activation key SUSE SSH Test Key x86_64 has been created" text
+
```

Comment:
Please ` Copyright (c) 2010-2020 SUSE LLC` on top of file.

Reply:
Copyright should start with the year of publication of the work (this file in this case).  The copyright cannot go backward in time before the existence of the work.